workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -339,7 +339,7 @@ protected static string CreateNewWorkerId(string workerType)
         }
 
 
-        private void OnDisconnected(string reason)
+        protected virtual void OnDisconnected(string reason)
```

Comment:
instead of making this protected, just register to the event directly on the worker: 

e.g. `Worker.OnDisconnect +=ConnectionScreenController.OnDisconnected `

Reply:
Makes sense